CONTRIBUTING.md

```
@@ -0,0 +1,257 @@
+## 🔄 Development Workflow
+
+### 1. Create a Feature Branch
+
+Always work on a new branch, never on `main` or `dev`:
+
+```bash
+git checkout -b feature/your-feature-name
+# or
+git checkout -b fix/your-bug-fix
+```
+
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)